spot_rl_experiments/configs/config.yaml

```diff
@@ -19,6 +19,9 @@ WEIGHTS_TORCHSCRIPT:
   # Semantic place
   SEMANTIC_PLACE: "weights/semantic_place/sp12lhb6v10_12_ckpt.199.torchscript"
 
+  # Semantic place EE
+  SEMANTIC_PLACE_EE: "weights/semantic_place_ee/sp_jt_ckpt.16.torchscript"
```

Comment:
Semantic Place EE weights aren't relevant to this PR, so let's remove it for now. We can add this in the PR for adding EE skills